crates/services/importer/src/importer.rs

```diff
@@ -223,12 +232,55 @@ where
             .seal_block(&block_id, &result.sealed_block.consensus)?
             .should_be_unique(&expected_next_height)?;
 
+        // Update the total tx count in chain metadata
+        let total_txs = db_after_execution
+            // Safety: casting len to u64 since it's impossible to execute a block with more than 2^64 txs
+            .update_tx_count(result.sealed_block.entity.transactions().len() as u64)?;
```

Comment:
Related to about question about `usize` vs `u64`. Why is this being converted to `u64`?

Reply:
I'll normalize this to u64, mainly because usize is unpredictable.

Reply:
In this case, I think the conversion is appropriate since there's no way to get the length of an array as anything but a usize.